bpf/bpf_lxc.c

```diff
@@ -419,16 +420,13 @@ static __always_inline int handle_ipv6_from_lxc(struct __ctx_buff *ctx, __u32 *d
 	verdict = policy_can_egress6(ctx, tuple, SECLABEL, *dst_id,
 				     &policy_match_type, &audited, ext_err, &proxy_port);
 
+	if (verdict == DROP_POLICY_AUTH_REQUIRED)
+		verdict = auth_lookup(SECLABEL, *dst_id, node_id, (__u8)*ext_err);
 	/* Create CT entry if drop for auth required. */
-	if (verdict == DROP_POLICY_AUTH_REQUIRED) {
-		if (ct_status == CT_NEW) {
-			ct_state_new.src_sec_id = SECLABEL;
-			ct_create6(get_ct_map6(tuple), &CT_MAP_ANY6, tuple, ctx,
-				   CT_EGRESS, &ct_state_new, proxy_port > 0, from_l7lb,
-				   true);
-		} else if (!ct_state->auth_required) {
-			verdict = CTX_ACT_OK; /* allow if auth done */
-		}
+	if (verdict == DROP_POLICY_AUTH_REQUIRED && ct_status == CT_NEW) {
+		ct_state_new.src_sec_id = SECLABEL;
+		ct_create6(get_ct_map6(tuple), &CT_MAP_ANY6, tuple, ctx,
```

Comment:
Do we still need to create connection tracking entries with the new approach?

Reply:
Ah, at a glance it looks like these are now being removed by the last revert commit :+1: